### DISCUSSION TITLE
SU2 Unsteady NACA0012 Tutorial case

### DISCUSSION THREAD

**Discussion by Brimp** [su2code/SU2#1460] [Q&A]
Hello, Everyone.

Currently, I'm learning to perform unsteady simulation on SU2. I run successfully with the SU2 given tutorial unsteady NACA0012 case in compressible part. However, that case is the one which two restart file are given to performing the unsteady simulation. My question is that how to run the this unsteady case without pre-given restart file. I have tried RESTART_SOL = NO , but the results from this simulation is totally different with the given results as shown in the SU2 website. Specifically, there is no lift or drag coefficient periodic oscillation with time iteration. Can anyone tell me the normal process for performing general unsteady simulation in SU2 and take this SU2 tutorial NACA-0012 case as an example to explain. Thanks very much.

**Comment by TobiKattmann** [comment]
Hi just to check, how many Time iterations are you running? It can take quite a bit of time until the flow instabilities shine through and you get the periodic oscillations. So if not already the case, take your shedding period / Time-iterations-per-period and multiply that by (idk) 100 and let it run, and see whether that helps. 

Sometimes it helps to artificially "tickle" the simulation by adding some flow instability (e.g. change the Boundary conditions for a few iterations)... but that is not super straightforward todo.

let me know whether that helps

**Reply by pcarruscag** [reply]
Can also run it with a large time-step initially so that interesting things develop.
